### PR TITLE
add java arm performance post

### PR DESCRIPTION
**Pull request by bdemers**
This blog post has the following features:

- [ ] A GitHub Repository with a polished README
- [ ] A title that's approved by Dev Marketing
- [ ] A URL that's approved by Dev Marketing
- [ ] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_posts/2020-08-05-java-arm-performance.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+tags: [java, performance, arm, intel, benchmarks ]
+tweets:
+- "Arm wrestling #Java: Arm vs x86_64 performance 💪"
+- "#Java performance benchmarks for #Arm ⏩"
+- "You might be able to save money running your #Java app on #Arm💰"
+image: blog/java-arm-performance/java-performance-social.png
+type: awareness
+---
+
+Arm processors have been in the news lately, and it's causing confusion and worries about processor performance for some folks. After Apple announced its plan to switch to Arm-based processors, I heard people (incorrectly!) speculating the performance would be similar to a Raspberry Pi. Java on Arm is nothing new, but we are seeing increased Arm investment from cloud vendors. Amazon recently updated its ARM offerings, and Microsoft is working on porting the JVM to Arm64 for Windows (no doubt for future Azure support).
```
Comment (mraible) [inline _source/_posts/2020-08-05-java-arm-performance.md:17]: Arm > ARM

File: _source/_posts/2020-08-05-java-arm-performance.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+{% img blog/java-arm-performance/apache-spark-als.png alt:"Graph showing the Arm server performed better" width:"800" %}
+
+The Spark alternating least squares (ALS) benchmark is one of the few tests where both the Arm servers outpaced the t3.medium.
+
+{% img blog/java-arm-performance/apache-spark-bayes.png alt:"Graph showing the m6g was faster" width:"800" %}
+
+In the Spark Naive Bayes algorithm test, the m6g.medium was 8% faster than the t3.
+
+**Winner:** m6g.medium. this was almost too close to call, but in the words of Meat Loaf,  "[Two Out of Three Ain't Bad.](https://youtu.be/k5hWWe-ts2s)"
```
Comment (mraible) [inline _source/_posts/2020-08-05-java-arm-performance.md:53]: I think you have to embed this video! :) That way people don't leave the page and they can listen to music while they finish reading. Brilliant!

File: _source/_posts/2020-08-05-java-arm-performance.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+## Pricing
+
+At the end of the day, which system you pick may come down to a balance of price and performance. The on-demand pricing for an m6g instance (medium, large, xlarge, 2xlarge) was about 8.5% cheaper than the corresponding t3 instance.
+
+**Winner:** m6g.medium
+
+## Conclusion
+
+The overall winner of these benchmarks is my MacBook Pro! Joking aside, the difference between Amazon's second-generation Arm processors and the _equivalent_ Intel processor wasn't what I expected when I started writing this post. If I had to pick between t3.medium and the m6g.medium, I'd say the overall winner of this shadow is the m6g.medium.
```
Comment (mraible) [inline _source/_posts/2020-08-05-java-arm-performance.md:102]: overall winner of this shadow?

Comment (bdemers) [reply _source/_posts/2020-08-05-java-arm-performance.md:102]: DOH! good catch

**Pull request merged**